pages/index.tsx

```diff
@@ -25,7 +25,7 @@ export default function IndexPage({ pageProps, setReleaseNote }) {
     pluginNames[Math.floor(Math.random() * pluginNames.length)];
   const tagline = `Stop writing code to sync data to ${pluginName}*`;
 
-  useEffect(() => setReleaseNote(releaseNote), []);
+  useEffect(() => setReleaseNote(releaseNote), [releaseNote, setReleaseNote]);
```

Comment:
I don't think I understand re-triggering the useEffect hook when the setter (`setReleaseNote`) changes - what would cause that?

Reply:
Because the function comes from props, the `react-hooks/exhaustive-deps` says it should be in there.